catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/LookSensorValuesInterpretationTest.java

```diff
@@ -94,8 +103,7 @@ public void testLookSensorValues() throws InterpretationException {
 		assertEquals(LOOK_ROTATION, lookRotateFormula.interpretDouble(testSprite), DELTA);
 
 		Formula lookZPositionFormula = getFormulaBySensor(Sensors.OBJECT_LAYER);
-		assertEquals(testSprite.look.getZIndex(), lookZPositionFormula.interpretInteger(testSprite).intValue()
-				+ Constants.Z_INDEX_NUMBER_VIRTUAL_LAYERS);
+		assertEquals(1, lookZPositionFormula.interpretInteger(testSprite).intValue());
```

Comment:
Please use a double here. `interpretInteger` uses [Double.html#intValue()](https://docs.oracle.com/javase/7/docs/api/java/lang/Double.html#intValue()) which casts the double to an int by removing numbers after the decimal point. So `1.3` → `1` and so the test may miss a bug.
```suggestion
		assertEquals(1.0, lookZPositionFormula.interpretDouble(testSprite));
```